### PR TITLE
Solution for second quest

### PR DESCRIPTION
**Pull request by malmat94**
Solution for second quest

### CODE REVIEW

File: learning/basics/task2/MM-if_statement.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+        second_number = input(f"Now give me second number {name}: ")
+        self.__int_checker__(second_number)
+
+        print(f"{name}, Your result of adding for {first_number} and {second_number} is {int(first_number) + int(second_number)}!!!")
+
+    def __name_checker__(self):
+
+        if self.name == "":
+            print("All right Nameless Noob, keep Your secrets...")
```
Comment (Artur-Ws) [inline learning/basics/task2/MM-if_statement.py:63]: LOVE THAT ❤️

File: learning/basics/task2/MM-if_statement.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+            self.name = "Nameless Noob"
+
+        else:
+            pass
```
Comment (Artur-Ws) [inline learning/basics/task2/MM-if_statement.py:67]: You don't need to put `else:` if it should only pass

**Review by Artur-Ws — CHANGES_REQUESTED** [review]
You do the same stuff in `multiplier` method and in `adder` method, maybe better solution will be get the numbers one time, for example in `main_process`, then just use them in `adder` and `multiplier` methods - According to DRY principle, there will be much less code without repetitions :)

**Comment by malmat94** [comment]
Yo, Artur, is there still anything wrong?

**Review by Artur-Ws — APPROVED** [review]

**Pull request merged**